spec/paper_trail/compatibility_spec.rb

```diff
@@ -14,7 +14,7 @@ module PaperTrail
 
       context "when incompatible" do
         it "writes a warning to stderr" do
-          ar_version = ::Gem::Version.new("7.2.0")
+          ar_version = ::Gem::Version.new("8.0.0")
```

Comment:
Should this be 7.3 in line with  the other changes?
```suggestion
          ar_version = ::Gem::Version.new("7.3.0")
```

Reply:
It's a pretty interesting question.

I think no because there is `>` method in other places to compare versions. But in this place we have exact version to test with.

At the first version of this pull request, I used `7.3` because of these commits in the rails repository [here](https://github.com/rails/rails/commit/07d51f9fa3a05386010b640bd121da88c9e341e4) and [here](https://github.com/rails/rails/milestones). But I changed it to 8.0 because of [this comment](https://github.com/paper-trail-gem/paper_trail/pull/1485#discussion_r1718292221).

So, I think that we should use 7.3 in cases with `>`.